numpy/lib/tests/test_histograms.py

```diff
@@ -78,6 +78,10 @@ def test_density(self):
         assert_array_equal(a, .1)
         assert_equal(np.sum(a * np.diff(b)), 1)
 
+        # Test that passing False works too
+        a, b = histogram(v, bins, density=False)
```

Comment:
This depends on Deprecation being an error?

Reply:
Deprecation is always an error in our tests - that's why tests have to be changed whenever a deprecation is introduced.

Reply:
Only when `pytest.ini` is present. Probably good enough.